webpack.config.js

```diff
@@ -81,16 +82,26 @@ const allPlugins = [
   }),
 
   // Use BrowserSync.
-  new BrowserSyncPlugin({
-    host: 'localhost',
-    port: 3000,
-    proxy: proxyUrl,
-    files: [
-      {
-        match: ['wp-content/themes/**/*.php', 'wp-content/plugins/**/*.php'],
-      },
-    ],
-  }),
+  new BrowserSyncPlugin(
+    {
+      host: 'localhost',
+      port: 3000,
+      proxy: proxyUrl,
+      files: [
+        {
+          match: [
+            'wp-content/themes/**/*.php',
+            'wp-content/plugins/**/*.php',
+            `${themePublicPath}styles/*.css`,
```

Comment:
`css`, or `scss` doesn't matter?

Reply:
You are watching the build file not assets file. So yes css here is correct. This is a pathe to public folder

Reply:
Oh, cool :+1: